### PR TITLE
Fix badge on README for tests

### LINKED ISSUE #2599: Fix tests badge

### PR DESCRIPTION
**Pull request by gsheni**
- Closes #2599

### CODE REVIEW

**Comment by machineFL** [comment]
No linked issues found. Please add the corresponding issues in the pull request description. <br/> [Use GitHub automation to close the issue when a PR is merged](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)

File: README.md [hunk 1]
```diff
@@ -6,8 +6,8 @@
 </p>
 
 <p align="center">
-    <a href="https://github.com/alteryx/featuretools/actions?query=branch%3Amain+workflow%3ATests" target="_blank">
-        <img src="https://github.com/alteryx/featuretools/workflows/Tests/badge.svg?branch=main" alt="Tests" />
+    <a href="https://github.com/alteryx/featuretools/actions/workflows/unit_tests_with_latest_deps.yaml" alt="Tests" target="_blank">
```
Comment (gsheni) [inline README.md:9]:
```suggestion
    <a href="https://github.com/alteryx/featuretools/actions/workflows/tests_with_latest_deps.yaml" alt="Tests" target="_blank">
```

File: README.md [hunk 2]
```diff
@@ -6,8 +6,8 @@
...
+        <img src="https://github.com/alteryx/featuretools/actions/workflows/unit_tests_with_latest_deps.yaml/badge.svg?branch=main" alt="Tests" />
```
Comment (gsheni) [inline README.md:10]:
```suggestion
        <img src="https://github.com/alteryx/featuretools/actions/workflows/tests_with_latest_deps.yaml/badge.svg?branch=main" alt="Tests" />
```

File: README.md [hunk 3]
```diff
@@ -6,8 +6,8 @@
 </p>
 
 <p align="center">
-    <a href="https://github.com/alteryx/featuretools/actions?query=branch%3Amain+workflow%3ATests" target="_blank">
-        <img src="https://github.com/alteryx/featuretools/workflows/Tests/badge.svg?branch=main" alt="Tests" />
+    <a href="https://github.com/alteryx/featuretools/actions/workflows/tests_with_latest_deps.yaml" alt="Tests" target="_blank">
+        <img src="https://github.com/alteryx/featuretools/actions/workflows/tests_with_latest_deps.yaml/badge.svg?branch=main" alt="Tests" />
```
Comment (thehomebrewnerd) [inline README.md:10]: When I follow this image link, it still shows "no status" for me. Should this be working? 

image

Comment (gsheni) [reply README.md:10]: No, not yet

**Review by thehomebrewnerd — APPROVED** [review]

**Pull request merged**